llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -561,8 +561,8 @@ bool lowerEsimdConstructs(module_split::ModuleDesc &MD) {
 IrPropSymFilenameTriple saveModule(module_split::ModuleDesc &MD, int I,
                                    StringRef IRFilename = "") {
   IrPropSymFilenameTriple Res;
-  StringRef Suffix =
-      MD.isDoubleGRF() ? "_esimd_x2grf" : (MD.isESIMD() ? "_esimd" : "");
+  StringRef Suffix = MD.isDoubleGRF() ? MD.isESIMD() ? "_esimd_x2grf" : "_x2grf"
```

Comment:
Some parenthesis might make this more readable..maybe?

Reply:
yeah good point, let me move this into a seperate function and rework it a little